### ISSUE TITLE
set npm publishConfig registery in configuration

### ISSUE THREAD

**Issue by iMilad** [eirslett/frontend-maven-plugin#827]
Hi,

Is it possible to set the "publishConfig" registry url for uploading artifact in pom.xml rather than in package.json?

I have two different <profile> which I want to set for each one of them different artifactory path to push/upload.

**Comment by eirslett** [comment]
I don't think so... or maybe you can do it with an `--argument` when running npm publish?
This plugin isn't really meant for publishing npm packages - if that's your use case, it's a special one!

**Comment by tripleplayj** [comment]
Hi,

I have a multi-module project which I am using this plugin to do `npm publish`, while specifying the "publishConfig" in the package.json. When running locally this works and publishes a package to npm, however, the status is [ERROR]. Is there any additional documentation on how to achieve this or why the [ERROR] would be showing? Also having issues with security while running this in a pipeline. Any feedback is appreciated.
